app/assets/stylesheets/preact/sidebar-widget.scss

```diff
@@ -73,20 +73,25 @@
         }
       }
     }
-    .write {
+    .widget-image {
+      margin: auto -16px; // offset the padding of the widget for img full width.
```

Comment:
Just thinking out loud that we may need to set a width on this for future campaigns, but we can improve as we start adding more campaigns

Reply:
I thought of this as well (haven't implemented because I haven't finished the work on the image overall)